Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ruby:2.5
+
+# throw errors if Gemfile has been modified since Gemfile.lock
+RUN bundle config --global frozen 1
```

Comment:
It makes sense to also run `bundle` from within the container, no?

Reply:
`RUN` runs things in the container before it is made into an image (the end result of a Dockerfile).